File: programacao/python/respostas/selecao/22.py
saldo = float(input("Digite o saldo: "))

if saldo <= 200.0:
    print("Nenhum credito")
elif saldo <= 1000.0:
    credito = saldo * 0.20
    print(f"Credito: {credito:.2f}")
elif saldo <= 2000.0:
    credito = saldo * 0.30
    print(f"Credito: {credito:.2f}")
elif saldo <= 10000.0:
    credito = saldo * 0.50
    print(f"Credito: {credito:.2f}")
else:
    credito = saldo * 1.0
    print(f"Credito: {credito:.2f}")

File: programacao/c/respostas/selecao/22.c
#include <stdio.h>

int main(void) {
    float saldo, credito;

    printf("Digite o saldo: ");
    scanf("%f", &saldo);

    if (saldo <= 200.0) {
        printf("Nenhum credito\n");
    } else if (saldo <= 1000.0) {
        credito = saldo * 0.20;
        printf("Credito: %.2f\n", credito);
    } else if (saldo <= 2000.0) {
        credito = saldo * 0.30;
        printf("Credito: %.2f\n", credito);
    } else if (saldo <= 10000.0) {
        credito = saldo * 0.50;
        printf("Credito: %.2f\n", credito);
    } else {
        credito = saldo * 1.0;
        printf("Credito: %.2f\n", credito);
    }

    return 0;
}